tests/test_bugzilla.py

```diff
@@ -177,21 +185,21 @@ def test_closed_bug_with_failure():
     result.assert_outcomes(1, 1, 1)
 
 
-def test_multiple_bugs_failure(testdir):
+def test_multiple_bugs_skip_1(testdir):
     testdir.makeconftest(CONFTEST)
     testdir.makepyfile("""
         import os
         import pytest
 
-        @pytest.mark.bugzilla('1', '2', '4')
+        @pytest.mark.bugzilla('1', '4', '2')
         def test_new_bug():
             assert(os.path.exists('/etcccc'))
     """)
     result = testdir.runpytest(*BUGZILLA_ARGS)
-    result.assert_outcomes(0, 0, 1)
+    result.assert_outcomes(0, 1, 0)
```

Comment:
why this test, has changed ?

Reply:
Copy paste from RHV fork, this was change  @pytest.mark.bugzilla('1', '4', '2') so the results as well.